elastic/integrations
# Extract "Ingress Controller" to a separate integration

## What does this PR do?



This PR extracts `ingress_controller` from the `nginx` to a separate integration.

## Checklist


## How to test this PR locally



Run system tests with defer cleanup option: `elastic-package test system -v --defer-cleanup=120s`,
navigate to dashboards to see access logs.

## Related issues


- https://github.com/elastic/integrations/issues/373

## Screenshots



![nginx-ingress-controller-access-logs](https://user-images.githubusercontent.com/14044910/101501879-8f67d880-3970-11eb-88c4-8d0cc86f1aa1.png)
![nginx-ingress-controller-overview](https://user-images.githubusercontent.com/14044910/101501890-9262c900-3970-11eb-9a46-4fe6e28b77d8.png)
